src/js/OTSession.coffee

```diff
@@ -136,8 +139,11 @@ class TBSession
     objects = document.getElementsByClassName('OT_root')
     while( objects.length > 0 )
       e = objects[0]
-      if e and e.parentNode and e.parentNode.removeChild
-        e.parentNode.removeChild(e)
+      if e and e.removeChild
```

Comment:
I believe `this.resetElement` is doing the same

Reply:
Adjusted and used the helper method

Reply:
Sorry to be a pain @mark-veenstra but wouldn't `e.removeChild` require a node to be passed in. IMHO, I think you just have to see if `e` exists b/c `resetElement` will take care of the children

Reply:
@msach22 you are right. Changed the commit.